api/V1/recordingUtil.ts

```diff
@@ -63,6 +61,45 @@ export interface RecordingQuery {
   filterOptions: null | any;
 }
 
+// How close is a station allowed to be to another station?
+export const MAX_STATION_DISTANCE_THRESHOLD_METERS = 60;
+// The radius of the station is half the max distance between stations: any recording inside the radius can
+// be considered to belong to that station.
+export const MAX_DISTANCE_FROM_STATION_FOR_RECORDING = MAX_STATION_DISTANCE_THRESHOLD_METERS / 2;
+
+export function latLngApproxDistance(a: [number, number], b: [number, number]): number {
+  const R = 6371e3;
+  // Using 'spherical law of cosines' from https://www.movable-type.co.uk/scripts/latlong.html
+  const lat1 = (a[0] * Math.PI) / 180;
+  const costLat1 = Math.cos(lat1);
+  const sinLat1 = Math.sin(lat1);
+  const lat2 = (b[0] * Math.PI) / 180;
+  const deltaLng = ((b[1] - a[1]) * Math.PI) / 180;
+  const part1 = Math.acos(sinLat1 * Math.sin(lat2) + costLat1 * Math.cos(lat2) * Math.cos(deltaLng));
+  return part1 * R;
+}
+
+async function tryToMatchRecordingToStation(recording: Recording): Promise<StationId | null> {
```

Comment:
Can't we use the database to do this or are you trying to get more accurate?

Reply:
I think a database query made sense when we were wanting to do "live" matching of recordings to stations, but since we're just baking the results in when we add or update a recording, it's simpler to do it in client code.  This is the approach used by leaflet.js that I'm using for distance between two lat/lng points.